NeoML/src/Dnn/Layers/QualityControlLayer.cpp

```diff
@@ -32,8 +32,6 @@ void CQualityControlLayer::Reshape()
 	CheckArchitecture( inputDescs.Size() == 2, GetName(), "layer expects 2 inputs" );
 	CheckArchitecture( inputDescs[0].ObjectCount() == inputDescs[1].ObjectCount(), GetName(),
 		"Object count mismatch between inputs" );
-	CheckArchitecture( inputDescs[0].ObjectSize() == inputDescs[1].ObjectSize(), GetName(),
```

Comment:
Let's add this `CheckArchitecture` to the other derivatives of `CQualiltyControlLayer`. Don't wanna lose this diagnostics...
E.g. `CConfusionMatrixLayer::Reshape` (and others)

Reply:
Other derivatives already have input NeoAssert on it.